src/ProgressOnderwijsUtils/Html/HtmlFragmentSanitizeExtension.cs

```diff
@@ -55,8 +55,8 @@ public bool AllowAttribute(HtmlAttribute attr)
             static readonly Regex
                 SafeStyleRegex = new Regex(@"^
                     (\s*margin(-(left|right|top|bottom))?\s*:\s*\d+(px|em|cm|mm|)\s*)
-                    | (\s*color\s*:\s*\w+\s*)
-                    | (\s*font-weight\s*:\s*\w+\s*)
+                    | (\s*color\s*:\s*[^;]+)
+                    | (\s*font-weight\s*:\s*[^;]+)

```

Comment:
Is het hier ook nodig om alle karakters toe te staan? Zoals ik zo snel in de spek zag kan je wat woorden gebruiken of getallen: https://htmldog.com/references/css/properties/font-weight/. Dat zou je ook eenvoudig helemaal uit kunnen specificeren.

Reply:
Inderdaad wel mooi meegenomen